clinica/pipelines/t1_freesurfer_atlas/t1_freesurfer_atlas_utils.py

```diff
@@ -23,43 +27,69 @@ def compute_atlas(
     ):
         hemisphere = Path(path.stem).stem
         atlas_name = Path(path.stem).suffix[1:].split("_")[0]
-        sub, ses = to_process_with_atlases[1].split("_")
-        subject_dir = (
-            caps_directory
-            + "/subjects/"
-            + sub
-            + "/"
-            + ses
-            + "/t1/freesurfer_cross_sectional"
-        )
+        if "long" not in to_process_with_atlases[1]:
+            sub, ses = to_process_with_atlases[1].split("_")
+            fake_dir = (
```

Comment:
why "fake" dir?

Reply:
I named it fake_dir because it is fake in my mind. but it's more a "substitute" dir. I'll rename it  !

Reply:
Ok. Maybe add a comment as well for further reference on why we need this substitute directory.